lib/web_ui/test/golden_tests/golden_success_smoke_test.dart

```diff
@@ -10,6 +10,7 @@ import 'package:web_engine_tester/golden_tester.dart';
 
 void main() {
   test('screenshot test reports success', () async {
+    html.document.body.style.fontFamily = 'Roboto';
```

Comment:
Is Roboto available in Cirrus? I've been using `Arial` instead, which is available in many places (but still more predictable than `sans-serif`).

Reply:
Nope:
```
00:01 +0 -1: screenshot test reports success [E]                                                                                                                                                       
  Golden file smoke_test.png did not match the image generated by the test.
  (0.7266% of pixels were different. Maximum allowed rate is: 0.2800%).
  You can view the test report in your browser by opening:
  https://api.cirrus-ci.com/v1/artifact/task/5994192492298240/web_engine_test/test_results/smoke_test.report.zip
```

Reply:
@ferhatb here's another confirmation that Roboto isn't available in all Chromes. I'll change code to bundle it. We have it under txt/third_party

Reply:
YAY!